README.md

```diff
@@ -101,10 +101,10 @@ build type when running CMake (by setting the `CMAKE_BUILD_TYPE`
 variable), and the generated files then build that configuration.
 
 An example of a single-configuration build system are Makefiles. You can
-build the Release configuration of Uncrustify (from the build folder) with:
+build the Debug configuration of Uncrustify (from the build folder) with:
```

Comment:
How about `You can build either the Release or Debug configuration...` and then:
```
$ cmake -DCMAKE_BUILD_TYPE=Release ..
$ make

$ cmake -DCMAKE_BUILD_TYPE=Debug ..
$ make
```

This makes explicit that both versions are available.

Reply:
OK, I added both configs to that section.